chart/develop/templates/deployment.yaml

```diff
@@ -0,0 +1,299 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: svc-questionnaires
+  namespace: {{ .Values.namespace }}
+  labels:
+    worker: "no"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: svc-questionnaires
+      tier: web
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0%
+  template:
+    metadata:
+      labels:
+        app: svc-questionnaires
+        track: stable
+        tier: web
+    spec:
+      imagePullSecrets:
+        - name: {{ .Values.gitlabAuthSecretName }}
+        - name: {{ .Values.dockerAuthSecretName }}
+      containers:
+        - name: svc-questionnaires
+          imagePullPolicy: Always
+          image: docker.io/researchableuser/sdv-svc-questionnaires:{{ .Values.tag }}
+          resources:
+            limits:
+              memory: "1.5Gi"
+            requests:
+              memory: "768Mi"
+          env:
+          - name: RAILS_ENV
+            value: production
+          - name: RACK_ENV
+            value: production
+          - name: NODE_ENV
+            value: production
+          - name: REVISION
+            value: {{ .Values.commitSha }}
+          - name: APPSIGNAL_APP_ENV
+            value: {{ .Values.appsignalAppEnv }}
+          - name: APPSIGNAL_ACTIVE
+            value: "true"
+          - name: PAPERTRAIL_PROGRAM
+            value: questionnaires
```

Comment:
Since we want to get rid of papertrail, we can optionally wrap this in an if-statement so we can enable/disable this stuff from values.yaml. Optional though, we could always add this later.

Reply:
I'll remove it altogether

Reply:
Fixed.